### ISSUE TITLE
Issue with inserting data to tables with specific table names

### ISSUE THREAD

**Issue by wolfig** [questdb/questdb#2920]
### Describe the bug

I am working on a project using QuestDB as a backend. I insert data into this backend via a custom Apache Camel Java component employing the InfluxDB Line Protocol. When inserting data, I see issues when using specific table names. In certain cases, the table is created with the specific name, but the insertion fails, i.e. the table is never populated with data. If I change the table name and execute the exact same code, the table gets created and the data is inserted into the table. I do not see exception messages when data insertion fails. To the best of my knowledge, I do follow the naming restrictions described in the API documentation (https://questdb.io/docs/reference/api/ilp/overview/)

Examples:

* zz, zzquest, zzquestdb, zzquestdbroute, zzquestdbroutetes, test123456789123456789 work as expexted
* zzquestdbroutetest, Zzquestdbroutetes, zZquestdb fail

Note: I did not check if this outside of my JAVA environment.

### Expected Behavior

I expect the ILP to accept any table name following the name restrictions given in the documentation. This means the use of capital letters and strings of "any" length are allowed.

### Environment

```markdown
- **QuestDB version**: 6.6.1
- **QuestDB Database server runs on Debian 9
- Java Development environment:
- **OS**: Rocky Linux 9
- **Java 11
- **Browser**: Firefox 102.6.0esr
```

**Comment by sreenath-tm** [comment]
Can you confirm if the input is failing or if it is success but just that post input you are not able to see the data on the table?

**Comment by wolfig** [comment]
> Can you confirm if the input is failing or if it is success but just that post input you are not able to see the data on the table?

I'm not sure how I could verify this. What I can say is that the way the data is inserted into the tables is always the same, independent of the table name. I always check the result using the QuestDB web-client

**Comment by sreenath-tm** [comment]
I tried creating the table with the name specified and also inserted some data into it and both are working as expected. It would be helpful if you can provide screenshots of the issue you are facing.

**Comment by eugenels** [comment]
Hey @wolfig 
I cannot reproduce it. You might be experiencing a case sensitivity issue that was resolved in the 6.7 release. 
Could you update and test it with the new version?

**Comment by eugenels** [comment]
Cool! Thanks for updating this!
Fixed in 6.7.
